.github/workflows/kairos-ubuntu-22-lts.yml

```diff
@@ -0,0 +1,48 @@
+---
+name: Build Kairos Ubuntu 22 LTS
+
+on: # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - main
+    paths:
+      - images/kairos-ubuntu-22-lts/**
+  pull_request:
+    branches:
+      - main
+    paths:
+      - images/kairos-ubuntu-22-lts/**
+
+jobs:
+  build-container:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - uses: earthly/actions-setup@v1.0.8
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          version: "0.7.1"
```

Comment:
We'll want to make sure renovate can update this version of earthly

Reply:
The existing GitHub workflow also has this issue; let's file an issue and fix both?